flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/log/CygnusLogger.java

```diff
@@ -76,7 +112,14 @@ public void warn(String componentName, String message) {
      * @param message
      */
     public void debug(String componentName, String message) {
-        debug("[" + componentName + "] " + message);
+        try {
+            debug("[" + componentName + "] " + message);
+        } catch (Exception e) {
+            if (shutdown) {
+                System.err.println("A problem with the logging system was found... shutting down Cygnus right now!");
```

Comment:
The `traceAndExit()` function was fixed in https://github.com/telefonicaid/fiware-connectors/commit/0893348c3335775f7d995af54c3394081db9e2b4